5_databases/week1/homework.sql

```diff
@@ -0,0 +1,60 @@
+-- Active: 1699628373418@@127.0.0.1@3306@hyf_lesson1
+
+-- 1.Find out how many tasks are in the task table
+SELECT count (*)
+FROM task;
+
+-- 2.Find out how many tasks in the task table do not have a valid due date
+SELECT count (*)
+FROM task
+WHERE due_date <= NOW();
+
+
+-- 3.Find all the tasks that are marked as done
+SELECT *
+FROM task
+WHERE status_id = 3;
+
+
+-- 4.Find all the tasks that are not marked as done
+SELECT *
+FROM task
+WHERE status_id != 3;
+
+
+-- 5.Get all the tasks, sorted with the most recently created first
+SELECT *
+FROM task
+ORDER BY created DESC;
+
+
+-- 6.Get the single most recently created task
+SELECT MAX (CREATED)
```

Comment:
This only returns the created field. Think more simple:


```
SELECT * 
FROM task 
ORDER BY created DESC LIMIT 1;
```

cool interpretation tho